CHANGELOG.md

```diff
@@ -0,0 +1,74 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
+and this project **only** adheres to the following _(as defined at [Semantic Versioning](https://semver.org/spec/v2.0.0.html))_:
+
+> Given a version number MAJOR.MINOR.PATCH, increment the:
+> 
+> - MAJOR version when you make incompatible API changes
+> - MINOR version when you add functionality in a backward compatible manner
+> - PATCH version when you make backward compatible bug fixes
+
+## [0.5.0] - 2024-11-XX
+
+This release includes a user-mapping plugin system that enables dynamic user-mapping between OpenID Connect users and iRODS users. It also includes improved security through token validation.
+
+User-mapping functionality from previous versions of the HTTP API is now provided through the user-mapping plugin system. See [Mapping OpenID Users to iRODS](https://github.com/irods/irods_client_http_api/tree/0.5.0?tab=readme-ov-file#mapping-openid-users-to-irods) for details.
+
+### Changed
+
+- Validate OIDC ID tokens in client mode (#107).
```

Comment:
should this be in the 'added' section?

Reply:
Will move it.

Reply:
After reading the changelog section descriptions again, I'm now thinking all validation commits should go under "Changed".

According to the changelog website:
- "Added" is for new features
- "Changed" is for changes in existing functionality

The validation commits change the behavior of the HTTP API, but for the user, there's no new functionality.

Thoughts?

Reply:
if validation is a feature, it was added.

if http api is doing auth differently... it is changed.

still feels like added to me... but your call.

Reply:
That's true. Dang.

Going to have to use more brain power now.

Reply:
@MartinFlores751 Where would you put the following issues given the descriptions at https://github.com/irods/irods_client_http_api/pull/351#discussion_r1840312028?
- 107
- 270
- 343